bogota-abril-2019/cyclomatic-complexity/JuanChaparro/unluckyDays.js

```diff
@@ -0,0 +1,13 @@
+function unluckyDays(year) {
+    var countUnlucky = 0;
+    var mDate = new Date(year, 0, 1);
```

Comment:
what does mDate represents?

Reply:
I usually use m to represent 'My', so in this case is 'My Date'.
I changed it to dummyDate to be more specific.